cmd/ocm-backplane/console/console.go

```diff
@@ -1309,3 +1365,23 @@ func (ce *dockerLinux) stopContainer(containerName string) error {
 func (ce *dockerMac) stopContainer(containerName string) error {
 	return generalStopContainer(DOCKER, containerName)
 }
+
+// podman-exist for Linux
```

Comment:
Just my 2 cents. Looks the `containerIsExist` method is implemented separately for each struct (`podmanLinux`, `podmanMac`, `dockerLinux`, `dockerMac`), is that a bit redundancy? How about only write the logic for containerIsExist once and defining the container engine string `podman`, `docker` as a property of the structs themselves?
```
type baseContainerEngine struct {
    engineType string
}

func (b *baseContainerEngine) containerIsExist(containerName string) (bool, error) {
    return generalContainerIsExist(b.engineType, containerName)
}

type podmanLinux struct {
    baseContainerEngine
}
```
So, in case we were to add another container engine in the future, is this structure more clear?

Reply:
This is a good suggestion @xiaoyu74 , I feel that we need refactor most of the code to handle proper CRI( container runtime interface) and may belong to this PR . I have seen some more duplicate methods in console and will create a card for that.